pandas/core/tools/numeric.py

```diff
@@ -137,6 +141,15 @@ def to_numeric(arg, errors="raise", downcast=None):
     if errors not in ("ignore", "raise", "coerce"):
         raise ValueError("invalid error value specified")
 
+    # Handle inputs of "date" type as objects
```

Comment:
umm what is this?

Reply:
The idea was to change the behavior of `pd.to_datetime` for date-like objects. Refer to this discussion: https://github.com/pandas-dev/pandas/issues/43280#issuecomment-909508445